### DISCUSSION TITLE
Adding a Search Bar to Radix UI Docs Website

### DISCUSSION THREAD

**Discussion by hueypov** [radix-ui/website#815] [Ideas]
### Overview

The proposed feature involves integrating a search bar into the Radix UI documentation website. Currently, the absence of a search functionality limits users' ability to quickly navigate and find relevant information within the extensive documentation. By implementing a search bar, users would benefit from a more efficient and user-friendly experience, enabling them to swiftly locate specific components, features, or documentation sections.

### Examples in other libraries or sites
Examples of similar search functionalities can be found in various documentation websites and libraries, such as Bootstrap, React, and Tailwind CSS. These platforms integrate search bars that significantly enhance users' ability to find specific information quickly within their documentation.

Bootstrap documentation search
React documentation search
Tailwind CSS documentation search

### Who does this impact? Who is this for?
This addition would significantly benefit developers, designers, and users exploring Radix UI's documentation. Developers and designers seeking information about Radix UI components, their functionalities, and implementation details would be able to swiftly find relevant content. Furthermore, users new to Radix UI would have an improved onboarding experience, facilitating their understanding and usage of the library through easy access to comprehensive documentation.

**Comment by benoitgrelard** [comment]
The Radix Primitives section has search, but I agree it would be good to add search to Radix Themes section too.

**Comment by Kenzo-Wada** [comment]
Is it ok if I make a PR to this issue?
